app/models/participant.rb

```diff
@@ -73,6 +73,33 @@ def topic_name
     end
   end
 
+  # send email to team's reviewers in case a new submission is made
+  def mail_assigned_reviewers
+    # Find review mappings for the work done by this participant's team
+    mappings = ResponseMap.where(reviewed_object_id: self.assignment.id,
+                                 reviewee_id: self.team.id,
+                                 type: 'ReviewResponseMap')
+    unless mappings.nil?
+      mappings.each do |mapping|
+        reviewer = mapping.reviewer.user
+        Mailer.sync_message(
```

Comment:
this mail creation code should be somewhere else. single responsibility principle.

Reply:
Not sure where to move this mail creation/delivery code. Similar code is handled in models, controllers, and helpers right now. Maybe all of it could be consolidated into mailer.rb eventually?
![image](https://user-images.githubusercontent.com/41524135/155435482-b0832faa-3010-4fdb-871e-306f53bc08a2.png)

Reply:
Never mind on this actually, I'll cut down on the actual creation lines in participant by making a new method in mailer_helper.rb. Then participant will still send the mail